### PR TITLE
Learn syncActive without making findBestSyncPeer call

### PR DESCRIPTION
**Pull request by cemozerr**
## PR Description
Only check the syncActive boolean onTick events, do not create a SyncStatus. Also, change the default timeout seconds because acceptance tests are failing due to our slow startup.

### CODE REVIEW

File: sync/build.gradle [hunk 1]
```diff
@@ -27,7 +27,6 @@ dependencies {
     integrationTestImplementation testFixtures(project(':util'))
 
     integrationTestImplementation 'org.mockito:mockito-core'
-//    integrationTestImplementation 'org.hyperledger.besu:plugin-api'
```
Comment (mbaxter) [inline sync/build.gradle:30]: Did you intend to comment this out here?

**Review by mbaxter — APPROVED** [review]

File: sync/build.gradle (hunk 1 above)
Comment (cemozerr) [reply sync/build.gradle:30]: Yes, I'm actually deleting that line I commented but forgot to delete in a previous PR.

Comment (mbaxter) [reply sync/build.gradle:30]: Should we just delete instead of commenting-out then?

Comment (cemozerr) [reply sync/build.gradle:30]: That's what I'm doing here :D

Comment (mbaxter) [reply sync/build.gradle:30]: oh - lol 😂 read that wrong

**Pull request merged**